time-calculator/time_calculator.py

```diff
@@ -0,0 +1,72 @@
+def add_time(start_time, stop_time, day_of_week=None):
+    PM = "PM"
+    [intial_time, start_day_unit] = get_time(start_time)
+    [start_hours, start_minutes] = get_hours_minutes(intial_time)
+
+    [stop_hours, stop_minutes] = get_hours_minutes(stop_time)
+
+    stop_minutes = into_minutes(stop_minutes, stop_hours)
+
+    if start_day_unit == PM:
+        start_hours = start_hours + 12
+
+    start_minutes = into_minutes(start_minutes, start_hours)
+    total_minutes = start_minutes + stop_minutes
+
+    final_minutes = total_minutes % 60
+    total_hours = int(total_minutes / 60)
+
+    if final_minutes < 10:
+        final_minutes = "0" + str(final_minutes)
+    else:
+        final_minutes = str(final_minutes)
```

Comment:
`final_minutes` should be a number and we should do this formatting while calculating full time string